notebooks/ISS_Pipeline_-_Breast_-_1_FOV.py

```diff
@@ -13,18 +13,19 @@
 
 # EPY: START code
 import numpy as np
-import pandas as pd 
+import pandas as pd
 import matplotlib.pyplot as plt
 from showit import image, tile
 import pprint
 # EPY: ESCAPE %matplotlib inline
 
 from starfish.io import Stack
+from starfish.constants import Indices
 
 s = Stack()
-s.read('https://dmf0bdeheu4zf.cloudfront.net/ISS/fov_001/experiment.json')
+s.read('https://dmf0bdeheu4zf.cloudfront.net/20180606/ISS/fov_001/experiment.json')
 # s.squeeze() simply converts the 4D tensor H*C*X*Y into a list of len(H*C) image planes for rendering by 'tile'
-tile(s.squeeze());  
+tile(s.squeeze());
```

Comment:
How do extra spaces get into the .py files? Did I do this? :|

Reply:
Automated tooling.